src/transformers/modeling_tf_utils.py

```diff
@@ -312,9 +312,10 @@ def booleans_processing(config, **kwargs):
 
     if tf.executing_eagerly():
         # Pure conv models (such as ConvNext) do not have `output_attentions`
-        final_booleans["output_attentions"] = kwargs.get("output_attentions", None)
-        if final_booleans["output_attentions"] is None:
-            final_booleans["output_attentions"] = config.output_attentions
+        if "output_attentions" in kwargs:
+            final_booleans["output_attentions"] = (
+                kwargs["output_attentions"] if kwargs["output_attentions"] is not None else config.output_attentions
+            )
```

Comment:
The previous version was passing down `final_booleans["output_attentions"]=False` in pure conv models, which would set the `output_attentions` argument to `False`. The new version results in no argument, which is the desired behavior.

Reply:
Can you add a comment that `"output_attentions"` will be in `kwargs`, with a value of `None` if unset? That change made me pause for a couple of minutes.